### ISSUE TITLE
Can't maximize/restore window by doubleclicking on titlebar 

### ISSUE THREAD

**Issue by Winterwolf** [Javier-DarthPalpatine/firefox-hide-caption-titlebar-plus#313]

**Comment by vovcacik** [comment]
Possibly related to #312

**Comment by Javier-DarthPalpatine** [comment]
* I've commented & posted a FIXED version in related issue #312. Please let me know if there is any further issue in this.

Thanks!
J

**Comment by Winterwolf** [comment]
Thank you, but still not working. I can move window, but doubleclick doing nothing (Linux Mint 17.3 Cinnamon).

**Comment by Javier-DarthPalpatine** [comment]
Okp, Tks!
I've tested this prerelease ver only on Windows 7 64, I'll try on Linux.
Could you tell me if in Firefox for Linux standard, dbl-clicking works e.g. in tab-bar background? or in another place?

**Comment by Winterwolf** [comment]
No. Double click doesn't work in tab-bar even without plugin. Only in title-bar.

**Comment by vovcacik** [comment]
@Winterwolf so if I understand you 3.0.8pre3 fixed this for you:
- dragging of custom HCTP titlebar
- double click on custom HCTP titlebar

But you still have issues with
- double click in tab-bar

What exactly are you double clicking in tabbar? If you got the same issue 
> even without plugin
why don't you file bug report at [https://bugzilla.mozilla.org/](https://bugzilla.mozilla.org/)?

**Comment by Winterwolf** [comment]
Maybe we bad understand each other (because English is not my native language, sorry for that). I mean that I installed your plugin because I wanted to Firefox looks in Linux same as in Windows, to merge tab-bar and title-bar, and your plugin helps me as well, thank you so much! But in Linux tab-bar hasn't any double click functionality (this is not a bug). I can maximize/restore windows without plugin by doubleclicking on native title-bar (not tab-bar), but can't do it with new, merged bar :)

**Comment by vovcacik** [comment]
All kudos go to @Javier-DarthPalpatine :-)

So there is still some problem with double clicking. Post screenshot please, and mark the area you're trying to double click.

**Comment by Winterwolf** [comment]
When I double clicking at green colored area - window maximized/restored, red colored areas don't react.
At first picture it's ok, but at second not.

![f1](https://cloud.githubusercontent.com/assets/405685/14249008/50a5480a-fa80-11e5-8b67-e2e0d54f2a04.png)

![f2](https://cloud.githubusercontent.com/assets/405685/14248987/3cdde78c-fa80-11e5-9d87-9c0de4111ca5.png)

**Comment by vovcacik** [comment]
I see, yeah that should work, at least it does on Windows (tested with Firefox 47.0a2 and HCTP 3.0.5).

So this issue is not related to #312 as I originally thought. The issue as you describe wasn't subject of #312, nor is fixed by 3.0.8 pre-release version introduced there.

@Javier-DarthPalpatine any ideas?

**Comment by trlkly** [comment]
On my Ubuntu-based Linux (tahrpup 6.0.5) I don't just not get it moved, but it actually crashes Firefox! But this is on Firefox Developer Edition with E10s turned on. 

But it might be the same problem, just with different results. So I thought I'd mention it here, instead of starting my own bug

**Comment by jonilam** [comment]
I'm also facing the same issue as @Winterwolf. I'm using Kubuntu 16.04 and Firefox 46.0.1. This issue exist also in HCTP 3.0.8pre3.

**Comment by Javier-DarthPalpatine** [comment]
I can confirm that this issue belongs to **Firefox-for-LINUX itself**,  in Windows it's working.
